src/js/options.js

```diff
@@ -298,31 +321,93 @@ var SettingsTab = {
     }
   },
 
-  saveImport: async function () {
+  saveSearches: async function (data) {
     try {
-      var parsedImport = JSON.parse(
-        document.getElementById("settings_json").value
-      );
+      var parsedData = JSON.parse(data);
 
       if (
         confirm(
           "Are you sure you want to override your local settings with these values?"
         )
       ) {
-        await ConfigFile.parseJSONFile(parsedImport, true);
+        await ConfigFile.parseJSONFile(parsedData, true);
 
         // Update UI according to this change
         mainConfigurationUpdated();
 
         Notiflix.Notify.Success("New configuration saved");
+        return true;
       }
     } catch (err) {
       console.error(err);
       Notiflix.Notify.Failure(
-        "Configuration couldn't be saved. Please verify that the file is valid."
+        "Configuration couldn't be saved. Please verify that the file/data is valid."
       );
+      return false;
+    }
+  },
+
+  importFromFile() {
+    document.getElementById("settings_fileInput").click();
+  },
+
+  fileImported(event) {
+    let selectedFile = _.first(this.files);
+    var reader = new FileReader();
+
+    reader.onload = function (event) {
+      SettingsTab.saveSearches(event.target.result);
+    };
+    reader.readAsText(selectedFile);
+
+    // Reset the value
+    event.target.value = "";
+  },
+
+  exportToFile: async function () {
```

Comment:
The "Export" button currently saves the configuration automatically, but it should ask the user _where_ to save the file.

> Ref: `Add a "Export to file" button that will open the system file browser so the user can select the folder and the file name were to save his current settings.`

Reply:
This is totally weird. It is working fine on my system. It asks for a location where you want to save file but not in your as well as Saidmurod mac. And we (Saidmurod) share the same chrome version. 

<img width="1282" alt="Screenshot 2021-05-12 at 4 39 27 PM" src="https://user-images.githubusercontent.com/11942435/117965665-a7d58800-b340-11eb-97c8-c9e106e86d0d.png">

Reply:
_Maybe_ it's because because you have your browser configured to always ask for where save files, while Saidmurod and me have configured to always put files on the same folder. If that's the case (you can try it by just changing your Download settings on Chrome), then you can leave it as it's.

Reply:
Yes, you must have configured to download file directly. That's why it's happening on your system.

<img width="703" alt="custom_download" src="https://user-images.githubusercontent.com/11942435/118222866-06644880-b49e-11eb-8f0a-e86ee1996aad.png">